crates/floresta-wire/src/p2p_wire/mod.rs

```diff
@@ -87,3 +87,4 @@ pub mod running_node;
 pub mod socks;
 pub mod stream_reader;
 pub mod sync_node;
+pub mod tests;
```

Comment:
Don't add this here for now. It creates like a thousand warnings

Reply:
Done!!